schemas/cna/null-asset-config-1.yml

```diff
@@ -0,0 +1,12 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /cna/null-asset-config-1.yml
+  addr_block:
+    type: string
```

Comment:
Should this also have

```
required:
- "$schema"
```
?

Reply:
a type that will to be used inline and crossrefed can't require the schema field otherwise we would need to define an override like this

```yaml
  - provider: null-asset
    identifier: my-null
    overrides:
      schema: /cna/null-asset-config-1.yml
      addr_block: bla
```

Reply:
this can be observed also here https://github.com/app-sre/qontract-schemas/blob/3803f49c99dc0319226d5455c0d7ddd841881ce2/schemas/app-sre/saas-file-target-1.yml

Reply:
But isnt that also true for the other 2 configs [aws-assume-role-config-1.yml](https://github.com/app-sre/qontract-schemas/pull/319/files#diff-f7a9830e1528f42d08fa0d8460a8b0f874d21cdb6481f9a37b38a8d516f04f87) and [aws-rds-config-1.yml](https://github.com/app-sre/qontract-schemas/pull/319/files#diff-e1c191c537f117368689cc81263cd9f575a08eb36217ac8f42208741db3c2e0e) ? I do not understand why they are handled differently here?

Reply:
that was unintentional and basically a bug that would have surfaced when we would have started using overrides for those two asset types.

fixed in ff793ba

good catch!